### PR TITLE
PDFmorph manual PDF and HTML

### LINKED ISSUE #24: Make pdfmorph web-page
critical for release

### LINKED ISSUE #26: Make PDFmorph user manual

### LINKED ISSUE #27: Package PDFmorph tutorial files
Make HTML and PDF manual modeled after [PDFgui HTML manual](https://www.diffpy.org/doc/pdfgui/index.html) and [PDFgui PDF manual](https://www.diffpy.org/doc/pdfgui/pdfgui.pdf) with sources [here](https://github.com/diffpy/diffpy.pdfgui/tree/master/doc/manual)

### LINKED ISSUE #30: add links to documentation and webpage in README

### LINKED ISSUE #39: Add a link to the project web-page at the top of the GitHub repo
e.g., see here for an example:
https://github.com/regro/regolith

### PR DESCRIPTION
**Pull request by Sparks29032**
When complete, this should close #24, close #26, and close #27. Can also close #30 and close #39 in this PR after the manual is finished.

@sbillinge I have commented changes to our current `.rst` files (added references to the user manuals).
You do not have to check the changes in the `.texinfo` files as I have compiled it into a [PDF](https://github.com/diffpy/diffpy.pdfmorph/blob/9e0a4bba58f7bd382833288f6f37464f2ad6f042/doc/manual/pdfmorph.pdf).
If you have time, I would like some feedback on section 4 and what in that section is okay to include in the paper proper.

User manual todo:

- [x] Acknowledgements
- [x] Disclaimer and redistribution
- [x] Introduction
- [x] Installation
- [x] Tutorials
- [x] Theory behind morphs (isotropic expansion, temperature as independent assumption, shape prefactor)
- [x] Summary of options in reference guide

Website todo (requires merging PR #120 and #121 and doing a pull):

- [x] Add links to webpage and manual on `README.rst`
- [x] Edit webpage landing and quickstart so they refer users to the manual first

### CODE REVIEW

**Comment by sbillinge** [comment]
I am happy to merge these fixes without pre-commit passing, but we need to figure out the right configuration of things to get this right.  there seems to be a conflict between black and flake8.  i left a comment on one of @caden's prs.  here ae some recommendations:
https://github.com/psf/black/blob/06ccb88bf2bd35a4dc5d591bb296b5b299d07323/docs/guides/using_black_with_other_tools.md#flake8

i also thought it is better to go with 88 instead of 115 for the line length which will make abit more work, but good to standardize I think

**Comment by Sparks29032** [comment]
Opened PR #118 to fix `pre-commit` errors.

File: doc/manual/acknowledgements.texinfo [hunk 1]
```diff
@@ -0,0 +1,15 @@
+The PDFmorph app was developed by the Billinge group at Columbia University.
+The examples in the tutorial part of the manual were made possible by data
+collected by Soham Banerjee at Brookhaven National Laboratory,
+Benjamin Frandsen during his PhD at Columbia Unversity,
+and Long Yang from Tonji University.
+
+The PDFmorph development team is the set of all contributors to the PDFgui project.
```
Comment (sbillinge) [inline doc/manual/acknowledgements.texinfo:7]: pdfmorph project?

**Review by sbillinge — COMMENTED** [review]
one small tweak?  Or maybe just leave it as "diffpy" project and we won't have to update it for all the different packages?

File: doc/manual/acknowledgements.texinfo (hunk 1 above)
Comment (Sparks29032) [reply doc/manual/acknowledgements.texinfo:7]: Yep, sorry referenced the `PDFgui` blurb

File: README.rst [hunk 2]
```diff
@@ -107,8 +107,8 @@ version using ::
 USING PDFmorph
 ------------------------------------------------------------------------
 
-For detailed instructions and full tutorial, consult our
-`online documentation <https://www.diffpy.org/diffpy.pdfmorph/index.html>`_.
+For detailed instructions and full tutorial, consult the user manual
+on our `website <www.diffpy.org/diffpy.pdfmorph/>`.
```
Comment (Sparks29032) [inline README.rst:111]: Reference user manual in `README.rst`.

File: doc/manual/source/quickstart.rst [hunk 3]
```diff
@@ -4,7 +4,8 @@ PDFmorph Tutorial
 #################
 
 Welcome! This will be a quick tutorial to accquaint users with PDFmorph
-and some of what it can do.
+and some of what it can do. For a more detailed tutorial, check out
+our :download:`user manual <../pdfmorph.pdf>`.
```
Comment (Sparks29032) [inline doc/manual/source/quickstart.rst:8]: Reference user manual in `quickstart.rst`.

File: doc/manual/source/index.rst [hunk 4]
```diff
@@ -35,7 +35,8 @@ the plotted PDFs.
 Finally, we note that though ``PDFmorph`` should work on other spectra 
 that are not PDFs, it has not been extensively tested beyond the PDF.
 
-To get started, please go to the :ref:`quick_start`.
+To get started, please download our :download:`user manual  <../pdfmorph.pdf>`
+or visit the :ref:`quick_start`.
```
Comment (Sparks29032) [inline doc/manual/source/index.rst:39]: Reference user manual in `index.rst`.

File: doc/manual/acknowledgements.texinfo [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+The DiffPy development team is the set of all contributors to DiffPy projects.
+Each member of the development team maintains copyright on their individual
+contributions to the code base. For a detailed contribution history of
+@code{PDFmorph}, see the git logs at @url{https://github.com/diffpy/diffpy.pdfmorph}.
+
+For more information about the application, please visit
+@url{https://www.diffpy.org/diffpy.pdfmorph}.
+Report any bugs to diffpy-users@@googlegroups.com or submit
+an issue to @url{https://github.com/diffpy/diffpy.pdfmorph/issues}.
```
Comment (Sparks29032) [inline doc/manual/acknowledgements.texinfo:15]: Edited per in-line comment.

**Pull request merged**